### PR TITLE
add login/register functionality

### PR DESCRIPTION
**Pull request by Unomars4**

### CODE REVIEW

File: src/Components/DashboardPage/Login-Register/Login.tsx [hunk 1]
```diff
@@ -12,9 +14,14 @@ const Login = ({isOpen}: ILoginForm) => {
...
-    const handleLogin = (event: React.SyntheticEvent): void => {
+    const handleLogin = async (event: React.SyntheticEvent): Promise<void> => {
         event.preventDefault();
-        console.log({email: email.value, password: pw.value, remember: rememberpw.value});
+        const userData = {
+            email: email.value,
+            password: pw.value,
+        }
+        const loginRes = await login(userData);
+        console.log(loginRes);
```
Comment (ritzballares) [inline src/Components/DashboardPage/Login-Register/Login.tsx:24]: is this needed?

Comment (ritzballares) [inline src/Components/DashboardPage/Login-Register/Login.tsx:24]: is `console.log(loginRes)` needed here?

File: src/Components/DashboardPage/Login-Register/Register.tsx [hunk 2]
```diff
@@ -11,10 +12,12 @@ const Register = () => {
...
     const lastName: IUseInputField = useInputField("text");
 
-    const handleRegister = (event: React.SyntheticEvent): void => {
+    const handleRegister = async (event: React.SyntheticEvent): Promise<void> => {
         event.preventDefault();
         if (pw.value === confirmpw.value) {
-            console.log({email: email.value, password: pw.value, lastName: lastName.value, firstName: firstName.value});
+            const registerData = {email: email.value, password: pw.value, lastName: lastName.value, firstName: firstName.value};
+            const registerRes = await register(registerData);
+            console.log(registerRes);
```
Comment (ritzballares) [inline src/Components/DashboardPage/Login-Register/Register.tsx:20]: is the log still needed?

**Review by ritzballares — APPROVED** [review]
Are logs still needed for the return value from login/register? Otherwise, looks good to me.

File: src/Components/DashboardPage/Login-Register/Login.tsx (hunk 1 above)
Comment (Unomars4) [reply src/Components/DashboardPage/Login-Register/Login.tsx:24]: I'll remove it just had it there for testing.

**Pull request merged**